team-props/static-analysis/lint-config.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+
+  <issue id="MissingTranslation" severity="ignore" />
+  <issue id="GoogleAppIndexingWarning" severity="ignore" />
+
+  <issue id="OldTargetApi" severity="informational" />
+
+  <!-- Ignore RtL warnings since the app doesn't support RtL -->
```

Comment:
DownloadManager might be used in an app that does support RtL no?

Reply:
This only applies to this codebase, client applications would need to reference this `lint-config` explicitly for it to be picked up by their buildscript. At least, that's how I understand it 😄